ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpRequestRetry.kt

```diff
@@ -189,7 +189,7 @@ public class HttpRequestRetry internal constructor(configuration: Configuration)
         ) {
             delayMillis = {
                 if (respectRetryAfterHeader) {
-                    val retryAfter = response?.headers?.get(HttpHeaders.RetryAfter)?.toLongOrNull()
+                    val retryAfter = response?.headers?.get(HttpHeaders.RetryAfter)?.toLongOrNull()?.times(1000)
```

Comment:
What if it's a date?

Reply:
We don't support a date for now, because there is no implementation of Http Date parser available for multiplatform.